packages/server/src/queue/queue.service.ts

```diff
@@ -22,10 +22,9 @@ import { AlertsService } from '../alerts/alerts.service';
  */
 @Injectable()
 export class QueueService {
-  constructor(
-    private connection: Connection,
-    private alertsService: AlertsService,
-  ) {}
+  readonly MAX_QUESTIONS_PER_QUEUE = 30;
```

Comment:
30 will probably be too low for Scott's use case. Also, one of the things with my work is adjusting the system to allow students to have 2 questions (one "task/demo" question and one regular question). So, in a lab of maybe 30 people (I will double-check with Scott and Ramon on Wednesday if it can realistically be larger than this), each student can have 2 questions, so maybe a max of 60 could be better but that's also kinda crazy amount of questions. I'll see what Ramon and Scott say

Reply:
Agree, lets chat with Ramon and Scott about this. Allowing more queues can simply crash frontend based on my tests. Unless we implement pagination

Reply:
Ah I see yeah, we'll probably need to implement pagination at some point... eugh.